Makefile

```diff
@@ -30,35 +30,16 @@ DEVWORKSPACE_CTRL_SA=devworkspace-controller-serviceaccount
 INTERNAL_TMP_DIR=/tmp/devworkspace-controller
 BUMPED_KUBECONFIG=$(INTERNAL_TMP_DIR)/kubeconfig
 RELATED_IMAGES_FILE=$(INTERNAL_TMP_DIR)/environment
+GO_PACKAGE_PATH ?= $(shell head -n1 go.mod | cut -d " " -f 2)
+BUILD_TIME = $(shell date -u '+%Y-%m-%dT%H:%M:%SZ')
+REF = $(shell cat .git/HEAD | sed 's|ref: ||')
+GIT_COMMIT_ID := $(shell cat .git/${REF})
 
-ifeq (,$(shell which kubectl))
```

Comment:
Heh, sorry -  this was because i changed the order by splitting up our old Makefile. I added a default so that the help target is what gets called when you type `make` now.